### PR TITLE
Update onboard-public-officers-using-non-se-machines.md

### PR DESCRIPTION
**Pull request by denisecollard**
To indicate what GMD refers to

### CODE REVIEW

**Comment by denisecollard** [comment]
close

File: docs/log-into-techpass-portal.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Log in to TechPass Portal
```
Comment (SenthilKumarSivakami) [inline docs/log-into-techpass-portal.md:1]: Pls remove these changes as i have clubbed the log in to TechPass portal and edit profile content.

File: docs/manage-security-verification-for-wog-account.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Set up security verification for Whole-of-Government (WOG) account
```
Comment (SenthilKumarSivakami) [inline docs/manage-security-verification-for-wog-account.md:1]: pls remove these changes as this is now a step in the onboarding PO into TechPass

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 3]
```diff
@@ -1,16 +1,19 @@
-# Onboard public officers
+# Onboarding public officer using non-SE GSIB device
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:1]: pls remove this change as i have already updated based on Sowmya's review comments.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
 
-This article guides public officers to sign up for TechPass account and to onboard to it using their non-SE GSIB machine. They can use their TechPass account to access Singapore Government Technology Stack (SGTS) products and services from GSIB and GMD machines.
+This article guides public officers to sign up for TechPass account and to onboard to it using their non-SE GSIB machine.
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:3]: pls remove this change as i have already updated based on review comments.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 5]
```diff
@@ -1,16 +1,19 @@
...
 **Overview**
 
 Following are the steps involved in your TechPass onboarding journey.
 
 ?> Complete the below steps using your non-SE GSIB machine.
 
-## Step 1. Sign up for TechPass
 <details>
-  <summary>How to sign up for TechPass?</summary>
+  <summary>a. Sign up for TechPass</summary><br>
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:16]: Pls remove this change as this was updated based on the review comments.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 6]
```diff
@@ -22,23 +25,21 @@ Following are the steps involved in your TechPass onboarding journey.
 
   2. Enter your organisational email address and select **I'm not a robot**.
 
-  ?> Format of your organisational email address shall be *your_name<span>@</span>agency.gov.sg* or *your_name<span>@</span>tech.gov.sg*
+  ?> Format of your organisational email address shall be _your_name<span>@</span>agency.gov.sg_ or _your_name<span>@</span>tech.gov.sg_
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:28]: Pls remove this change.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 7]
```diff
@@ -22,23 +25,21 @@ Following are the steps involved in your TechPass onboarding journey.
...
 
   <kbd>![sign-up-submit](assets/images/onboarding/po-non-se/sign-up-submit.png)</kbd>
 
   3. Click **Submit**. An invitation will be sent to this email address.
 
-  ?> A TechPass account is created for you but this will be in pending state. This becomes activated once you complete the onboarding journey which is explained in **Accept invitation** and **Onboard to TechPass**.
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:31]: Pls remove this change.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 8]
```diff
@@ -22,23 +25,21 @@ Following are the steps involved in your TechPass onboarding journey.
...
+  ?> A TechPass account is created for you but this will be in pending state. This becomes activated once you complete the onboarding journey.
 
 
 </details>
 
-## Step 2. Accept invitation
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:36]: pls remove this change.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 9]
```diff
@@ -22,23 +25,21 @@ Following are the steps involved in your TechPass onboarding journey.
...
-
 <details>
-  <summary>Steps to accept invitation</summary>
+  <summary>b. Accept invitation</summary><br>
 
-  Public officer has to accept this invitation within 30 days to onboard to TechPass. Invitation is not valid after 30 days and you need to sign up again for a TechPass account.
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:41]: pls remove this change.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 10]
```diff
@@ -51,46 +52,26 @@ Following are the steps involved in your TechPass onboarding journey.
   <kbd>![accept-invitation](assets/images/onboarding/po-non-se/accept-invitation.png)</kbd>
 
 
-</details>
+  3. Proceed to [**set up security verification for your WOG account**](manage-security-verification-for-wog-account). This step is required if you intend to use TechPass account from your GMD.
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:55]: pls remove the change.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 11]
```diff
@@ -51,46 +52,26 @@ Following are the steps involved in your TechPass onboarding journey.
...
 
-## Step 3. Set up security verification for your WOG account
+</details>
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:57]: pls remove the change.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 12]
```diff
@@ -51,46 +52,26 @@ Following are the steps involved in your TechPass onboarding journey.
...
 
 <details>
+<<<<<<< Updated upstream
+  <summary>c. Onboard to TechPass</summary><br>
+=======
   <summary>How to set up security verification for WOG account?</summary>
 
-  Public officers need to set up security verification(multi-factor authentication) for their Whole-of-Government(WOG) account to access Singapore Government Technology Stack (SGTS) services and tools from their GMD device.
+  Public officers need to set up security verification(multi-factor authentication) for their Whole-of-Government(WOG) account to access Singapore Government Technology Stack (SGTS) services and tools from their internet Government Managed Devices(GMD).
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:65]: pls remove the change.

File: docs/onboard-public-officers-using-non-se-machines.md [hunk 13]
```diff
@@ -51,46 +52,26 @@ Following are the steps involved in your TechPass onboarding journey.
...
 
   If you do not have a GMD or do not intend to access SGTS services using GMD, skip this step and proceed to **Onboard to TechPass**.
 
   _To set up security verification for WOG account:_
+>>>>>>> Stashed changes
```
Comment (SenthilKumarSivakami) [inline docs/onboard-public-officers-using-non-se-machines.md:70]: pls remove this change

**Review by SenthilKumarSivakami — COMMENTED** [review]
@denisecollard I think all your changes in this PR seem to be reverting the latest changes done :-) . So pls cancel this PR and let me know what has to be updated so that i can update it directly.

File: docs/access-sgts-services-using-techpass.md [hunk 14]
```diff
@@ -1,15 +1,13 @@
-# Log in with TechPass
+# Log in with TechPass account to access SGTS services
 
-You can use your TechPass account to access SGTS services such as SEED, GCC2.0, Cloud File Transfer (CFT), SHIP-HATS and Developer portal from your GMD and GSIB devices.
+TechPass users can use their TechPass to securely and seamlessly access SGTS services such as SEED, GCC2.0, Cloud File Transfer (CFT), Container Stack (C Stack), SHIP-HATS and Developer portal using their GMD and GSIB devices.
 
-This article guides you on how to use TechPass to access SGTS resources.
+This article guides you on how to use TechPass to access SGTS resources (For this instance, through Developer portal).
```
[the remark this answers is not in the document]
Comment (denisecollard) [reply docs/access-sgts-services-using-techpass.md:5]: Noted

File: docs/_sidebar.md [hunk 15]
```diff
@@ -1,11 +1,11 @@
...
-- Getting started
-  - [Onboard public officer](onboard-public-officers-using-non-se-machines)
-  - [Onboard vendor](onboard-vendors-to-techpass)
-  - [Log in with TechPass](access-sgts-services-using-techpass)
-- Manage account
-  - [Edit Profile](edit-profile)
-  - Reset MFA
-    - [WOG account](reset-security-verification-for-wog-account)
-    - [TechPass account](reset-techpass-mfa-for-new-device)
+- Onboarding
```
[the remark this answers is not in the document]
Comment (denisecollard) [reply docs/_sidebar.md:2]: Noted

File: docs/access-sgts-services-using-techpass.md [hunk 16]
```diff
@@ -1,15 +1,13 @@
-# Log in with TechPass
+# Log in with TechPass account to access SGTS services
 
-You can use your TechPass account to access SGTS services such as SEED, GCC2.0, Cloud File Transfer (CFT), SHIP-HATS and Developer portal from your GMD and GSIB devices.
+TechPass users can use their TechPass to securely and seamlessly access SGTS services such as SEED, GCC2.0, Cloud File Transfer (CFT), Container Stack (C Stack), SHIP-HATS and Developer portal using their GMD and GSIB devices.
```
[the remark this answers is not in the document]
Comment (denisecollard) [reply docs/access-sgts-services-using-techpass.md:3]: Noted

**Comment by denisecollard** [comment]
Closing this PR

**Pull request closed without merging**